JS101/fundamentals/rock-paper-scissors-project.md

```diff
@@ -109,6 +109,10 @@ Submit a solution with a pull request to this [file](https://github.com/TheOdinP
 - [vartanbeno's solution](https://github.com/vartanbeno/rock-paper-scissors) - [View in browser](https://vartanbeno.github.io/rock-paper-scissors/)
 - [Alexey Leonov's solution](https://github.com/axelerleo/rock-skissors-paper) - [View in browser](https://axelerleo.github.io/rock-skissors-paper/)
 - [Fabio Oliveira's solution](https://github.com/ffabiorj/rock_paper_scissors) - [View in browser](https://cdn.rawgit.com/ffabiorj/rock_paper_scissors/dab5e260/index.html)
+<<<<<<< HEAD
```

Comment:
Hey @afeso, there seems to be a few merge conflict markers in this PR. Can you remove them please? 😄